src/App/App.tsx

```diff
@@ -42,6 +43,7 @@ function App() {
         <Route path='/getting-started/:userId' element={<Form onSubmit={handleFormSubmit} />} />
         <Route path='/results' element={<Results />} />
         <Route path='/log-in'  element={<LoginForm />} />
+        <Route path='/saved-budgets' element={<SavedBudgets />} />
         {/* <Route path='/:id/saved'  element={} />
         <Route path='/log-in'  element={} /> */}
```

Comment:
I wonder if the commented out routes are leftover from the previous iterations, in which case we can clean it up later.